src/Jackett.Common/Definitions/polishtracker-api.yml

```
@@ -43,6 +43,10 @@ settings:
     type: info
     label: About your API key
     default: "Find your API Key by accessing your <a href=\"https://pte.nu/\" target =_blank>PolishTracker</a> account <i>Settings</i> page and clicking on the <b>API</b> section."
+  - name: polish_language
+    type: checkbox
+    label: Add POLISH to title if has Polish language. Use this if you using Sonarr/Radarr
+    default: false
```

```suggestion
  - name: multilang
    type: checkbox
    label: Replace MULTI & PL by another language in release name
    default: false
  - name: multilanguage
    type: select
    label: Replace MULTI & PL by this language
    default: POLISH
    options:
      POLISH: POLISH
      MULTI.POLISH: MULTI.POLISH
```
in line with other `multi` replacements, e.g. https://github.com/Jackett/Jackett/blob/master/src/Jackett.Common/Definitions/polishsource.yml